desktop/main.js

```diff
@@ -406,7 +406,14 @@ const mainWindow = (() => {
                 browserWindow.webContents.send(ELECTRON_EVENTS.BLUR);
             });
 
-            app.on('before-quit', () => quitting = true);
+            app.on('before-quit', () => {
+                // Adding __DEV__ check because we only want to remove it on dev version
+                // https://github.com/Expensify/App/issues/15965#issuecomment-1483182952
+                if (__DEV__) {
+                    app.removeAsDefaultProtocolClient('new-expensify');
```

Comment:
We're using 'new-expensify' string 3 times throughout the app, let's define it in `CONST` instead